### DISCUSSION TITLE
Project Identity concerning closed source, paid and unmaintained projects

### DISCUSSION THREAD

**Discussion by Unkn0wnCat** [osmlab/awesome-openstreetmap#158] [General]
Hey everyone,

this is the discussion that started in #156.

### The Problem

The discussion started because of the request to add a project which is based on OSM, but is not open-source. This has been seen by some members of the community as too far from the original aim of the list. Whilst some have suggested the maintainer (me, currently) should decide it, I'd hate to just decide this on my own, as this project is meant to be a community project.

The problem we now face is how do we decide which projects to add and which to pass up on, as I see the issue with just having every project that uses OSM as a mapping backend on this list.

### My View

When I've inherited the list two years ago it used to look like [this](https://github.com/osmlab/awesome-openstreetmap/tree/2e00d0ec5d2cfa8869006690e8ea337fdcdb2410), there were a few dead/outdated projects and a bunch of unmerged pull-requests. Back then most projects were open-source, but it's noteworthy some projects on the list were also closed-source, but free. (See some of the tools, like [hdyc](http://hdyc.neis-one.org/))
Currently there are a few closed source projects on the list, which is bittersweet as those projects are very cool uses of OSM, but I acknowledge it is not in the spirit of the wider Awesome List ecosystem.

### What I need from you

Here's where you all come in. Let's come up with a few strict rules to review list entries by. We can use the voting reactions (:+1: / :-1:) and the comment functions on responses below to gauge which rules are in line with what the majority of the people here want.

### Timeline

In my eyes we should give this discussion about a month (1st of November 2024), but if there is stilly heavy discussion by then we can always extend this.

---

Please remember to be nice to each other - Let's make this list even more awesome together! 😸 

--- 

Reactions:
:+1: - I support this rule
:-1: - I don't support this rule
👀 - Projects not matching this rule should be kept on a seperate list

**Comment by Unkn0wnCat** [comment]
**only open-source projects (as per [OSI-Definition](https://opensource.org/osd))**

**Comment by Unkn0wnCat** [comment]
**only free (in terms of cost) projects**

**Comment by Unkn0wnCat** [comment]
**only projects using OSM as primary data source or destination**

**Reply by mnalis** [reply]
Voted yes, but I guess might be OK if it equally supports other sources (e.g. imaginary great routing app that allowed user to choose between OSM or Google data might be included provided it is both FOSS and gratis; it would just need to be somewhat more _Awesome_ to be included)
